frontend/src/ui/time.tsx

```diff
@@ -30,23 +36,39 @@ export const RelativeDate: React.FC<RelativeDateProps> = ({ date }) => {
     const MONTH = 30.5 * DAY;
     const YEAR = 365.25 * DAY;
 
+    const liveDate = ((time: number, unit: Intl.RelativeTimeFormatUnit) => {
+        const parts = intl.formatToParts(time, unit);
+        return String(parts[1].value) + parts[2].value;
+    });
+
     const prettyDate = (() => {
-        const intl = new Intl.RelativeTimeFormat(i18n.language);
+        let time: number;
+        let unit: Intl.RelativeTimeFormatUnit;
         if (secsDiff <= 55) {
```

Comment:
this could also be using switch case instead of if/else - not sure what is preferred or more readable

Reply:
Yeah, now that you mention it... switch-case could only work with an ugly trick like `switch(true)`, but doesn't make much sense here. 
I decided to go with the destructuring, but I'm honestly not sure which I like better or find more readable. 🤷